Source/WebKit/Platform/IPC/unix/AttachmentUnix.cpp

```diff
@@ -76,4 +81,14 @@ Attachment& Attachment::operator=(Attachment&& attachment)
 
 Attachment::~Attachment() = default;
 
+void Attachment::encode(Encoder& encoder) const
```

Comment:
I think Attachment::encode and Attachment::decode should be removed in your design.

Reply:
Yes, but the other patch does this for unix.
This patch removes them only for Darwin and Windows. See pr #4122